config.yml

```diff
@@ -1,584 +1,368 @@
 title: Write a Learning Lab course
 tagline: Learn how to author your own Learning Lab course.
-description: Get starter code, or let the Learning Lab bot guide you as you author your first course.
+description: Use Learning Lab's strengths for fast feedback to author your own course.
 template:
   name: lab-starter
-  repo: write-a-ll-course-template
+  repo: write-a-ll-course-template-2.0
```

Comment:
This is a placeholder, because the template course requires such a major change, it would break the currently launched course. It should definitely be updated once this branch is merged. I'll add a note to the original post of this PR.